index.html

```diff
@@ -44,5 +51,137 @@ <h2>I’am a Software Developer</h2>
         <a href="#"><img src="src/Social-Media.svg" alt="social-media" /> </a>
       </div>
     </section>
+
+    <section class="my-works">
+      <div class="works-section-a">
+        <h2>My Recent Works</h2>
+
+        <div class="wrapper">
+          <div class="Feature">
+            <img src="src/Img-Placeholder-n1.svg" alt="Feature stories" />
+            <h3>Multi-Post Stories</h3>
+            <p class="Feat-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="tech-stack-1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <p class="btn">See Project</p>
```

Comment:
- [ ] [OPTIONAL] Please implement this as a `button` or an `a` tag.
- [ ] [OPTIONAL] do the same for all other buttons.